### ISSUE TITLE
UUIDPrimaryKeyMixin not working with SQLite

### ISSUE THREAD

**Issue by floriankick** [MasoniteFramework/orm#453]
**Describe the bug**
The UUIDPrimaryKeyMixin ist not working properly with SQLite. 
When creating a Model, the UUID is generated and saved correctly in the database, but the models id is set to the rowid instead of the actual uuid.

**To Reproduce**
1. Setup project using SQLite database
2. Create Model with UUIDPrimaryKeyMixin 
2. Save new record by calling create() function, without providing a value for the id field:
`project = Project.create({'label': name})`
3. Access the id field:
`new_id = project.id`
4. It will be some integer value

**Expected behavior**
The id field of the model should be set to the UUID value that was saved to the database.

**What database are you using?**
 - Type: SQLite
 - Version: 3.34.0
 - Masonite ORM: 1.0.46

**Additional context**
I traced the problem down to the SQLitePostProcessor.
It returns the lastrowid in process_insert_get_id():
```
    def process_insert_get_id(self, builder, results, id_key="id"):
        # [...]

        results.update({id_key: builder.get_connection().get_cursor().lastrowid})

        return results
```

**Comment by floriankick** [comment]
Just saw, that the implementation for MySQL is the same. Maybe the bug occurs there as well.

**Comment by josephmancuso** [comment]
@floriankick yeah it looks like this was missed when we implemented the post processor. Thanks for reporting it, we will take a look

@girardinsamuel @Marlysson

**Comment by josephmancuso** [comment]
Just a thought we need to explore but maybe we can do this via registering an observer in the `UUIDPrimaryKeyMixin `

**Comment by josephmancuso** [comment]
@floriankick This is fixed with version 1.0.47
